crxmake.sh

```diff
@@ -13,13 +13,16 @@ name="augury"
 files="manifest.json build src images index.html frontend.html popup.html"
 
 crx="$name.crx"
-if [ $CIRCLE_BUILD_NUM ]; then
-  crx="$name-$CIRCLE_BUILD_NUM.crx"
-fi
-
 pub="$name.pub"
 sig="$name.sig"
 zip="$name.zip"
+
+# assign build name to zip and crx file in circleci env
+if [ $CIRCLE_BUILD_NUM ] || [ $CIRCLE_ARTIFACTS ]; then
```

Comment:
if we are in CircleCI we use the build number to label the crx and zip accordingly.